### PR TITLE
Sprint 5 gw

### PR DESCRIPTION
**Pull request by G-White-ISB**
Image search portal front end (currently at search.html) and ohif view (ohif.html) added for end-of-Feb prototype of MVP

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -32,6 +32,13 @@
     <link rel="icon" type="image/x-icon" href="{% static 'img/favicon.ico' %}" />
     <link type="text/css" rel="stylesheet" href="{% static 'css/style.css' %}" />
     <script type="text/javascript" src="{% static 'js/libs/modernizr.js' %}"></script>
+   <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+   <script src="https://code.jquery.com/ui/jquery-ui-git.js"></script> -->
+   <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+  <link rel="stylesheet" href="/resources/demos/style.css">
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+
```
Comment (s-paquette) [inline templates/base.html]: We'll want to add all libs into static/libs, because we don't want them changing out from under us. Additionally, we use RequireJS in numerous other files, so the names of the files need to be consistent. (They DO need to be normalized to give us less of a headache.) Eg. see base.js

Comment (s-paquette) [reply templates/base.html]: (Generally, if you want something to appear in all of the HTML and JS files, you can add it to the RequireJS config in base.js and it'll be available across the board, because base.js is pulled in by base.html, and that in turn is the basis for all of our templates.)

Comment (G-White-ISB) [reply templates/base.html]: So I am now using static libs only. However I had to source jquery and and jquery-ui directly even though they are already in base.js. Sourcing base.js did not work either.

Comment (s-paquette) [reply templates/base.html]: I'll have a look at that, but I suspect I know what the issue is. (Short version: we're using RequireJS, so everything has to be inside a require block which requires base.js.)
